<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

    <title>Nataša Tagasovska</title>

    <meta name="author" content="Natasa Tagasovska">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="images/favicon/favicon.ico" type="image/x-icon">
    <link rel="stylesheet" type="text/css" href="stylesheet.css">
    
  </head>

  <body>
    <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
      <tr style="padding:0px">
        <td style="padding:0px">
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr style="padding:0px">
              <td style="padding:2.5%;width:63%;vertical-align:middle">
                <p class="name" style="text-align: center;">
                  Nataša Tagasovska
                </p>
                <p>I'm a Principal Machine Learning Scientist and Scientific Manager at <a href="https://www.gene.com/scientists/our-scientists/prescient-design">Prescient Design, Genentech</a> in the Frontier Research team.
                  My interests have always revolved around ML methods for causal learning and uncertainty estimation, and recently I get to explore their capabilities within applications in computational biology.
                  </p>
                <p>
                  Prior to Prescient, I was a Sr. Data Scientist at the <a href="https://www.datascience.ch/">SDSC</a> at EPFL-ETHZ, where I worked on interdisciplinary projects, applying ML to physical and social science research efforts.
I hold a PhD in Information Systems (Data Analytics track) from University of Lausanne and a BS and MSC in Computer Science and Engineering. During my studies I interned at Facebook (Meta) AI Research and NATO.
                   </p>
                   <p>
                  Get in touch if you want to chat more about research or life at Prescient!
                </p>

                <p style="text-align:center">
                  <a href="mailto:natasa.tagasovska@roche.com">Email</a> &nbsp;/&nbsp;
                  <a href="data/Natasa_Tagasovska_Academic_CV_March24.pdf">CV</a> &nbsp;/&nbsp;
                  <a href="https://scholar.google.com/citations?hl=en&user=S2ZUSL0AAAAJ">Scholar</a> &nbsp;/&nbsp;
                  <a href="https://twitter.com/tagasovska?lang=en">Twitter</a> &nbsp;/&nbsp;
                  <a href="https://github.com/tagas/">Github</a>
                </p>
              </td>
              <td style="padding:2.5%;width:40%;max-width:40%">
                <a href="images/NatasaTagasovskaBW.jpeg"><img style="width:100%;max-width:100%;object-fit: cover; border-radius: 10%;" alt="profile photo" src="images/NatasaTagasovskaBW.jpeg" class="hoverZoomLink"></a>
              </td>
            </tr>
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
              <tr>
              <td style="padding:20px;width:100%;vertical-align:middle">
                <h2>Research</h2>
                <p>
                  At Prescient, my projects include OOD generalization and generative modeling of molecules, aiming at designs that conform with multiple properties.
                </p>
                 <p>
                  Some of my recent projects are listed below. For full publication list, see my <a href="https://scholar.google.com/citations?hl=en&user=S2ZUSL0AAAAJ">Google scholar.</a>
                </p>
              </td>

            </tr>
                        <tr>
              <td style="padding:20px;width:100%;vertical-align:middle">
                <h2>Activities</h2>
                <p>
<dl>
  <dt>Apr 2024 - Thrilled to be joining the <a href="https://www.wiml.org/"> WiML</a> Board of Directors</dt>
  <dt>Dec 2023 - General chair for the <a href="https://neurips.cc/virtual/2023/affinity-workshop/66602"> WiML Workshop at NeurIPS 2023</a> </dt>
  <dt>Dec 2023 - Program Chair at <a href="https://ai4d3.github.io/">New Frontiers of AI for Drug Discovery and Development NeurIPS 2023</a></dt>
  <dt>Nov 2023 - Co-organizing an <a href=https://ai.ethz.ch/details.industry-research-workshop-machine-learning-for-health-care-and-life-sciences.69673.html">Industry Research Workshop</a> at the ETH AI Center</dt>
  <dt>July 2023 - Associate Chair at <a href="https://icml.cc/Conferences/2023">ICML 2023 </a></dt>
</dl>
                </p>
              </td>

            </tr>
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>


    <tr >
      <td style="padding:20px;width:25%;vertical-align:middle">
        <div class="one">
          <img src='images/genentech_ab_bb.png' width=100%>
        </div>

      </td>
      <td style="padding:20px;width:75%;vertical-align:top">
        <a href="https://openreview.net/pdf?id=0cJ8ERfnrM">
          <span class="papertitle">Antibody DomainBed: Out-of-Distribution Generalization in Therapeutic Protein Design</span>
        </a>
        <br>
        <strong>Natasa Tagasovska*</strong>
		<a href="https://jiwoncpark.github.io/">Ji Won Park*</a>,
		<a href="https://mkirchmeyer.github.io/">Matthieu Kirchmeyer</a>,
		<a href="https://ncfrey.github.io/">Nathan C Frey</a>,
		<a href="https://scholar.google.com/citations?hl=en&user=zglcuwEAAAAJ">Andrew Martin Watkins</a>,
        <a href="https://scholar.google.com/citations?hl=en&user=VDhTJHUAAAAJ">Aya Abdelsalam Ismail</a>,
        <a href="https://jamasb.io/">Arian Rokkum Jamasb</a>,
        <a href="">Edith Lee</a>,
        <a href="">Tyler Bryson</a>,
        <a href="https://stephenra.com/">Stephen Ra</a>,
        <a href="https://kyunghyuncho.me/">Kyunghyun Cho</a>
        <p>
        Releasing a dataset of antibody-antigen binding energy emulating distribution shifts across active drug design cycles, and the accompanying backbones suitable for large molecules.
        </p>
      </td>
    </tr>
    <tr >
      <td style="padding:20px;width:25%;vertical-align:middle">
        <div class="one">
          <img src='images/botied.png' width=100%>
        </div>
      </td>
      <td style="padding:20px;width:75%;vertical-align:top">
        <a href="https://arxiv.org/abs/2306.00344">
          <span class="papertitle">BOtied: Multi-objective Bayesian optimization with tied multivariate ranks</span>
        </a>
        <br>
        <strong>Natasa Tagasovska*</strong>
		<a href="https://jiwoncpark.github.io/">Ji Won Park*</a>,
        <a href="https://scholar.google.com/citations?user=HlSrRJAAAAAJ&hl=en">Michael Maser</a>,
        <a href="https://stephenra.com/">Stephen Ra</a>,
        <a href="https://kyunghyuncho.me/">Kyunghyun Cho</a>
        <p>
        We show a natural connection between the non-dominated solutions and the highest multivariate rank. This motivates a new Pareto-compliant indicator and an acquisition function for multi-objective Bayesian Optimization.
        </p>
      </td>
    </tr>
          <tr >
      <td style="padding:10px;width:25%;vertical-align:middle">
        <div class="one">
          <img src='images/clues.png' width=100%>
        </div>
      </td>
      <td style="padding:20px;width:75%;vertical-align:top">
        <a href="https://openreview.net/pdf?id=5iryL0a6x4">
          <span class="papertitle">MoleCLUEs: Molecular Conformers Maximally In-Distribution for Predictive Models</span>
        </a>
        <br>
        <a href="https://scholar.google.com/citations?user=HlSrRJAAAAAJ&hl=en">Michael Maser</a>,
        <strong>Natasa Tagasovska</strong>
        <a href="https://scholar.google.com/citations?user=j76uSIwAAAAJ&hl=en">Jae Hyeon Lee</a>,
        <a href="https://scholar.google.com/citations?hl=en&user=zglcuwEAAAAJ">Andrew Martin Watkins</a>
        <br>
        <em>NeurIPS 2023 AI for Science Workshop</em>, 2023
        <p>
        Generating conformers that explicitly minimize predictive uncertainty for structure-based ML models.
        </p>
      </td>
    </tr>
          <tr >
      <td style="padding:20px;width:25%;vertical-align:middle">
        <div class="one">
          <img src='images/vcnn.png' width=100%>
        </div>
      </td>
      <td style="padding:20px;width:75%;vertical-align:top">
        <a href="https://proceedings.mlr.press/v206/tagasovska23a/tagasovska23a.pdf">
          <span class="papertitle">Retrospective Uncertainties for Deep Models using Vine Copulas</span>
        </a>
        <br>
        <strong>Natasa Tagasovska</strong>
        <a href="">Firat Ozdemir</a>,
        <a href="">Axel Brando</a>
        <em>AISTATS</em>, 2023
        <p>
        We compensate for the lack of built-in uncertainty estimates by supplementing any deep network, retrospectively, with a subsequent vine copula model that allows for retrospective estimation of aleatoric and epistemic uncertainty.
        </p>
      </td>
    </tr>
         <tr >
      <td style="padding:20px;width:25%;vertical-align:middle">
        <div class="one">
          <img src='images/svae.png' width=100%>
        </div>
      </td>
      <td style="padding:20px;width:75%;vertical-align:top">
        <a href="https://proceedings.mlr.press/v213/lopez23a/lopez23a.pdf">
          <span class="papertitle">Learning Causal Representations of Single Cells via Sparse Mechanism Shift Modeling</span>
        </a>
        <a href="">Romain Lopez*</a>,
        <strong>Natasa Tagasovska*</strong>
         <a href="https://stephenra.com/">Stephen Ra</a>,
        <a href="https://kyunghyuncho.me/">Kyunghyun Cho</a>
         <a href="https://web.stanford.edu/group/pritchardlab/home.html">Jonathan Pritchard</a>,
        <a href="https://g.co/kgs/3MkD8H9">Aviv Regev</a>
        <br>
        <em>CLeaR</em>, 2023
        <p>
        Modeling of gene expression profiles of cells under different genetic or chemical perturbations via sparse mechanishm shift.
        </p>
      </td>

    </tr>
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

          </tbody></table>
        </td>
      </tr>
    </table>
  </body>
</html>
